quantumapp/server.py

```
@@ -0,0 +1,248 @@
+
+def _read_json(environ) -> Dict[str, Any]:
+    try:
+        length = int(environ.get("CONTENT_LENGTH") or 0)
+    except ValueError:
+        length = 0
+    body = environ["wsgi.input"].read(length) if length > 0 else b""
+    if not body:
+        return {}
+    try:
+        return json.loads(body.decode("utf-8"))
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)